src/components/App.js

```diff
@@ -2,6 +2,8 @@ import React from 'react';
 import '../App.css';
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
+// eslint-disable-next-line no-unused-vars
```

Comment:
# [Optional]

Kindly make your PR Description more detailed enough to explain which logic has been  implemented